catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/controller/search/QueryRunnable.java

```diff
@@ -255,22 +254,13 @@ protected QueryResponse queryCatalog(String sourceId, SearchRequest searchReques
                 }
 
                 LOGGER.debug("Sending query: {}", query);
-                response = searchController.getFramework()
-                        .query(request);
+                ResultIterable results = new ResultIterable(searchController.getFramework(),
```

Comment:
❗️ We should keep using `searchController.getFramework().query(request)` here instead of switching to the new `ResultIterable`, otherwise we lose all the processing details returned by the `QueryResponse` that we used to handle in the catch blocks.